modules/database/src/ioc/dbStatic/dbLexRoutines.c

```diff
@@ -1257,7 +1257,7 @@ static void dbAlias(char *name, char *alias)
     DBENTRY dbEntry;
     DBENTRY *pdbEntry = &dbEntry;
 
-    if(dbRecordNameValidate(alias))
+    if(dbRecordNameValidate(alias) || dbRecordNameValidate(name))
```

Comment:
I think this would work.  I had been thinking more along the lines of checking that `pdbEntry->pflddes==NULL` after `dbFindRecord()`, but I think this would accomplish the same.

Reply:
Thanks Michael, yes, I assumed a parsing error was the right way to handle this